litmus/director/TCID-DIR-OP-CSTOR-POOL-RECOMMEND-CREATE-MIRROR/test.yml

```diff
@@ -41,6 +41,9 @@
           shell: | 
             kubectl get bd -n {{ namespace }}
 
+        - pause:	
```

Comment:
Just for future reference - can we apply some reconciliation logic here to retry on some API instead of pausing for 2 mins.

Reply:
We have already added reconciliation logic to check openebs is installed or not. This is a kind of mandatory pause that we need to invoke to make cstor pool creation process smooth. @harshshekhar15 .
Then also I will discuss this with Shovan and try to avoid this pause in ansible playbook. Thanks